src/test/unreferenced_file/specification/readme.md

```diff
@@ -1,4 +1,6 @@
 > see https://aka.ms/autorest
 
 ```yaml $(tag) == 'something'
+input-file:
```

Comment:
Could we create separate test for this? It would be nice to keep at least one test with empty `input-file`.

Reply:
should there be a separate folder under the "test" for each? I lumped them under the error name folder, but I can see that the separate folder might be better

Reply:
separate folder under `test/` for now.

Reply:
done